rust/README.md

```diff
@@ -0,0 +1,135 @@
+# Practical Networked Applications in Rust
+
+A training course on practical software construction in Rust.
+
+The final project in this course is a networked, persistent [key-value store]
+with multithreading, asynchronous I/O (via [tokio] and [tower]), a
+high-performance network protocol ([GRPC] via [prost] and [tower-grpc]),
+backed by a simple [LSM-tree] that the student will write themselves.
+
+Subjects covered include:
+
+- Structuring and maintaining Rust programs
+- Fun and foolproof parallel programming
+- Asyncronous programming with [futures] and [tokio]
+- High-performance networking with [GRPC] and [prost]
+- Benchmarking with [criterion] and [critcmp]
+- Robust and reliable error handling with [failure]
+- Serialization with [serde]
+- Build scripts and the interaction between build-time and run-time
+- Insights into inner workings of the language and libraries
+- How to learn what you don't know about Rust and find what you need to find
+- ... and etc.
```

Comment:
This line is extraneous, you already say "include" at the start.

Reply:
I'll remove it.